src/cache/redis_cacher.rs

```diff
@@ -18,6 +18,8 @@ pub struct RedisCache {
     pool_size: u8,
     /// It stores the index of which connection is being used at the moment.
     current_connection: u8,
+    /// It stores the max TTL for keys.
+    cache_ttl: u16,
```

Comment:
I think I would suggest not having the TTL value stored in the struct because we are going to use the TTL value only once when we are setting the cache in the Redis server. So by having the TTL value stored in the struct, it consumes unnecessary resources of the system. :slightly_smiling_face:

Reply:
> It is just two bytes which is really small and the same argument could be said for `threads` since it is only used once while initializing the actix server.

I think you are right let's have this then :slightly_smiling_face: .